tests/v1/user_deactivation_test.py

```diff
@@ -1,8 +1,8 @@
-import sys, os
-import warnings
+# import sys, os
+# import warnings
 
-warnings.filterwarnings("ignore", category=DeprecationWarning)
-sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '../..')))
+# warnings.filterwarnings("ignore", category=DeprecationWarning)
+# sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '../..')))
```

Comment:
revert this

Reply:
I am not the one who made those changes

Reply:
this beacause the `auth-v1` branch is not synced with `dev`

Reply:
Do I have to change my PR to `dev` ?

Reply:
it shows you made changes to the file
revert it on local
pull from dev branch and make sure your branch is updated